Observer/KlaviyoOauthObserver.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace Klaviyo\Reclaim\Observer;
+
+use Klaviyo\Reclaim\Helper\ScopeSetting;
+use Magento\Framework\Event\ObserverInterface;
+use Magento\Framework\Event\Observer;
+use Magento\Framework\Message\ManagerInterface as MessageManager;
+use Magento\Integration\Model\IntegrationFactory;
+use Magento\Integration\Model\AuthorizationService;
+use Magento\Integration\Model\OauthService;
+
+
+class KlaviyoOauthObserver implements ObserverInterface
+{
+    /**
+     * Klaviyo scope setting helper
+     * @var \Klaviyo\Reclaim\Helper\ScopeSetting $klaviyoScopeSetting
+     */
+    protected $_klaviyoScopeSetting;
+
+
+    /**
+     * @var IntegrationFactory $integrationFactory
```

Comment:
small nit, i guess it should follow
```suggestion
     * @var  \Magento\Integration\Model\IntegrationFactory $integrationFactory
```

Reply:
is that a standard or just to be consistent?  I am not seeing that pattern for the other Observers

Reply:
consistent

Reply:
consistent means I shouldn't have the fully qualified names since it isnt done like that for the other Observers

Reply:
Yes, the other observers haven't been edited in a while, we should convert them over, its better to provide the full path since its easier to distinguish where the protected variable is coming from rather than having to look in other places in the file